diff --git a/src/org/locationtech/jts/geom/Geometry.js b/src/org/locationtech/jts/geom/Geometry.js
index ee98617..d780839 100644
--- a/src/org/locationtech/jts/geom/Geometry.js
+++ b/src/org/locationtech/jts/geom/Geometry.js
@@ -1,27 +1,8 @@
-import WKTWriter from 'org/locationtech/jts/io/WKTWriter';
-import GeometryCollectionMapper from 'org/locationtech/jts/geom/util/GeometryCollectionMapper';
-import IllegalArgumentException from 'java/lang/IllegalArgumentException';
-import IsValidOp from 'org/locationtech/jts/operation/valid/IsValidOp';
-import InteriorPointArea from 'org/locationtech/jts/algorithm/InteriorPointArea';
-import UnaryUnionOp from 'org/locationtech/jts/operation/union/UnaryUnionOp';
-import SnapIfNeededOverlayOp from 'org/locationtech/jts/operation/overlay/snap/SnapIfNeededOverlayOp';
-import InteriorPointLine from 'org/locationtech/jts/algorithm/InteriorPointLine';
-import IsSimpleOp from 'org/locationtech/jts/operation/IsSimpleOp';
-import BufferOp from 'org/locationtech/jts/operation/buffer/BufferOp';
-import ConvexHull from 'org/locationtech/jts/algorithm/ConvexHull';
-import Centroid from 'org/locationtech/jts/algorithm/Centroid';
-import GeometryCollection from 'org/locationtech/jts/geom/GeometryCollection';
 import Comparable from 'java/lang/Comparable';
-import RelateOp from 'org/locationtech/jts/operation/relate/RelateOp';
-import InteriorPointPoint from 'org/locationtech/jts/algorithm/InteriorPointPoint';
 import Cloneable from 'java/lang/Cloneable';
 import Serializable from 'java/io/Serializable';
-import DistanceOp from 'org/locationtech/jts/operation/distance/DistanceOp';
-import Envelope from 'org/locationtech/jts/geom/Envelope';
-import RectangleContains from 'org/locationtech/jts/operation/predicate/RectangleContains';
-import Assert from 'org/locationtech/jts/util/Assert';
-import RectangleIntersects from 'org/locationtech/jts/operation/predicate/RectangleIntersects';
-import OverlayOp from 'org/locationtech/jts/operation/overlay/OverlayOp';
+import Envelope from './Envelope';
+import Assert from '../util/Assert';
 export default class Geometry {
 	constructor(...args) {
 		(() => {
@@ -452,4 +433,3 @@ Geometry.geometryChangedFilter = new (class {
 		geom.geometryChangedAction();
 	}
 })();
-
